ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/system.md

```diff
@@ -66,13 +66,13 @@ interface Management1
 system control-plane
    tcp mss ceiling ipv4 1344 ipv6 1366
    ip access-group acl4_1 in
+   ip access-group acl4_3 vrf default in
```

Comment:
technically this is not possible as this will be rendered as

```
inet-cloud(config-system-cp)#show run se system
system control-plane
   tcp mss ceiling ipv4 1344 ipv6 1366
   ip access-group acl4_3 in
   ip access-group acl4_2 vrf red in
   ip access-group acl4_2 vrf red_1 in
   ipv6 access-group acl6_3 in
   ipv6 access-group acl6_2 vrf blue in
   ipv6 access-group acl6_2 vrf blue_1 in
```

(cannot have both default and none defaut

Reply:
though we cannot do anything really about it - the sorting is good so will resolve